src/nextlabs_sdk/_cloudaz/_policy_search.py

```
@@ -1,27 +1,45 @@
+    "/console/api/v1/policy/search/savedlist",
+)
+_FIND_SAVED_SEARCH_SPEC = query_paginated(
+    SavedSearch,
+    "/console/api/v1/policy/search/savedlist/{name}",
+)
+
+_CRITERIA_ARG = "criteria"
+_SEARCH_ARG = "search"
 
 
 class PolicySearchService:  # noqa: WPS214
```

Ruff (0.15.21)

[warning] Invalid rule code in `# noqa`: WPS214

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)